File: LeetCode/kotlin/runningSum.kt
class Solution {
    fun runningSum(nums: IntArray): IntArray {
        var runningSum=mutableListOf<Int>()
        var sum=0
        for(i in nums.indices){
            sum=sum+nums[i]
            runningSum.add(sum)
        }
        return runningSum.toIntArray()
    }
}

File: LeetCode/swift/runningSum.swift
class Solution {
    func runningSum(_ nums: [Int]) -> [Int] {
        var runningSum=[Int]()
        var sum=0
        for i in 0..<nums.count{
            sum=sum+nums[i]
            runningSum.append(sum)
        }
        return runningSum
    }
}